engine/app/components/citizens_advice_components/footer.html.erb

```diff
@@ -5,7 +5,7 @@
         <div class="cads-grid-col">
           <p class="cads-footer__feedback">
             <%= link_to(
-              feedback.title,
+              feedback.title || t("citizens_advice_components.footer.website_feedback"),
```

Comment:
This fixes a deprecation warning with `ViewComponent::Translate` in view component 3